ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewHeaderFooterViewDemoController.swift

```diff
@@ -143,6 +161,12 @@ extension TableViewHeaderFooterViewDemoController: UITableViewDelegate {
         alert.addAction(action)
         present(alert, animated: true)
     }
+
+    private func forHeaderTapped(header: TableViewHeaderFooterView, section: Int) {
+        collapsedSections[section] = collapsedSections[section] != true
```

Comment:
How about just inverting the value?

`collapsedSections[section] = !collapsedSections[section]`